### PR TITLE
[Task] Update introduction text for list module and access control

### PR DESCRIPTION
**Pull request by tomwarwick**
[+] Move introduction paragraph into the first page of
the List Module chapter
[-]  Remove "filler" text from Access Control chapter

### CODE REVIEW

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 1]
```diff
@@ -1,6 +1,18 @@
 .. include:: /Includes.rst.txt
 .. _using-the-list-module-effectively:
 
+============
+Introduction
+============
+
+The list module allows you to browse through
```
Comment (brotkrueml) [inline Documentation/ListModule/UsingEffectively/Index.rst:8]:
```suggestion
The :guilabel:`Web > List` module allows you to browse through
```

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 2]
```diff
@@ -1,6 +1,18 @@
...
+each page or folder within your site and view all of the records that
+are stored within it. The List Module also gives you the ability to
```
Comment (brotkrueml) [inline Documentation/ListModule/UsingEffectively/Index.rst:10]:
```suggestion
are stored within it. The :guilabel:`List` module also gives you the ability to
```

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+create and manage records that don't have a dedicated Module.
```
Comment (brotkrueml) [inline Documentation/ListModule/UsingEffectively/Index.rst:11]:
```suggestion
create and manage records that don't have a dedicated module.
```

File: Documentation/ListModule/UsingEffectively/Index.rst [hunk 4]
```diff
@@ -1,6 +1,18 @@
...
+
+Sub pages, content elements and news stories are examples of the different
```
Comment (brotkrueml) [inline Documentation/ListModule/UsingEffectively/Index.rst:13]: Hmm, are we really talking about "news stories"? Not about "news articles" or "news records"?

Comment (tomwarwick) [reply Documentation/ListModule/UsingEffectively/Index.rst:13]: Yes

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**